LCM/dsc/engine/ca/CAInfrastructure/WebPullClient.c

```diff
@@ -1281,7 +1226,7 @@ MI_Result SetGeneralCurlOptions(CURL* curl,
 {
     CURLcode res;
     curl_easy_setopt(curl, CURLOPT_SSL_VERIFYHOST, 0L);
-    curl_easy_setopt(curl, CURLOPT_VERBOSE, 1L);
+    curl_easy_setopt(curl, CURLOPT_VERBOSE, 0L);
```

Comment:
>curl_easy_setopt [](start = 4, length = 16)

why we are changing the curl options?

Reply:
This verbose output goes to stderr causing problems in the wrapper python scripts. The data is not captured anywhere  else either.

---
In reply to: [263571706](https://github.com/Microsoft/PowerShell-DSC-for-Linux/pull/541#discussion_r263571706) [](ancestors = 263571706)